.github/workflows/dependabot_update_rbis_run.yml

```
@@ -0,0 +1,9 @@
+name: Update RBIs
+on:
+  pull_request:
+    branches:
+      - main
```

```suggestion
    types: [opened, synchronize]
    paths:
      - 'Gemfile.lock'
```

I don't think we need to filter for a specific branch.